src/app/services/course.service.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable } from '@angular/core';
+import { Course } from 'app-models';
+
+@Injectable({
+  // tslint:disable-next-line:no-unsafe-any
+  providedIn: 'root'
+})
+export class CourseService {
+  private static nextCourseId: number = 1;
+
+  constructor() {}
+
+  public getCourses(): Course[] {
```

Comment:
It would be better to use Obsrvable<Course[]> to simulate http call
```suggestion
  public getCourses(): Observable<Course[]> {
```